projects/ngneat/reactive-forms/src/lib/utils.ts

```diff
@@ -8,7 +8,7 @@ export function isFunction(x: any): x is Function {
   return typeof x === 'function';
 }
 
-export function isNil(v: any): boolean {
+export function isNil(v: any): v is null | undefined {
```

Comment:
Surely this should still be a Boolean type?

Reply:
Well, it was committed in this PR by mistaken, but in anyway it'd be better to use a typeguard instead of boolean. Reverted.